src/ext/ddtrace.h

```diff
@@ -5,6 +5,8 @@ extern zend_module_entry ddtrace_module_entry;
 
 ZEND_BEGIN_MODULE_GLOBALS(ddtrace)
 zend_bool disable;
+zend_bool ignore_missing_overridables;
+zend_bool lookup;
```

Comment:
How is this global variable `lookup` used? I only see usages of `class_lookup ` and `function_lookup `.

Reply:
Whoops, that wasn't supposed to go into this PR.